app/src/androidTest/java/com/github/polybooks/database/SaleDatabaseTest.kt

```diff
@@ -293,29 +295,58 @@ class SaleDatabaseTest {
         assertEquals(0,db.querySales().searchByTitle(saleTest.book.title).getCount().get())
     }
 
-
+    @Ignore
     @Test
```

Comment:
Are you sure you fixed the tests as explained by the commit title? The tests you modified (3 of them) seem to be `@Ignore`d

Reply:
Yes these three tests can be used to insert and delete sales from the firebase database. They always work (no asserts in them)